src/pubsub/streaming_subscription.rs

```diff
@@ -94,6 +94,7 @@ pub enum ModifyAcknowledgeError {
 pub struct AcknowledgeToken {
     id: String,
     channel: mpmc::Sender<UserAck>,
+    delivery_attempt: i32,
```

Comment:
I see that `api::ReceivedMessage::delivery_attempt` is also an `i32`, but does it ever make sense for this to be < 0?

Reply:
Potentially -- sometimes the google apis use -1 to denote a special value. The docs on this particular field say 0 is the only noted special value, but they could add another in the future.

I could see an argument for translating this into say Option<u32> or something more expressive. But it's simpler to just forward as-is, I don't think there's too much harm in this case